packages/cli/src/ActiveExecutions.ts

```diff
@@ -21,6 +24,11 @@ import { Logger } from '@/Logger';
 
 @Service()
 export class ActiveExecutions {
+	private queues = {
+		manual: new ConcurrencyQueue(config.getEnv('executions.manualConcurrency')),
+		others: new ConcurrencyQueue(config.getEnv('executions.concurrency')),
```

Comment:
Or is there a reason to keep it intentionally vague?

```suggestion
		production: new ConcurrencyQueue(config.getEnv('executions.concurrency')),
```